io/bufio.ts

```diff
@@ -211,8 +211,9 @@ export class BufReader implements Reader {
    * delim.
    * For simple uses, a Scanner may be more convenient.
    */
-  async readString(_delim: string): Promise<string | Deno.EOF> {
-    throw new Error("Not implemented");
+  async readString(delim: string): Promise<string | Deno.EOF> {
+    const buffer = await this.readSlice(delim.charCodeAt(0));
```

Comment:
Maybe throw if delim.length != 1

Reply:
Keeping in mind comments :point_down:, I'll check if `delim.lenght > 0`